projects/storefrontapp-e2e-cypress/cypress/integration/regression/express-checkout.e2e-spec.ts

```diff
@@ -0,0 +1,65 @@
+import * as checkout from '../../helpers/checkout-flow';
+
+context('Express checkout', () => {
```

Comment:
Better to split it into 3 tests:
- one to check if express redirects to first step when shipping address is missing
- second similar, but when payment details are missing
- third when everything is set
- fourth test - change delivery mode preferences (eg. to MOST_EXPENSIVE) and check that on review page

Also in third test do a check if on review page default shipping address and payment methods are used and proper delivery mode.